src/callback/BeforeMintCallbackERC721.sol

```diff
@@ -14,12 +14,13 @@ contract BeforeMintCallbackERC721 {
 
     /**
      *  @notice The beforeMintERC721 hook that is called by a core token before minting tokens.
+     *  @param _caller The address that calls the mint function.
      *  @param _to The address that is minting tokens.
      *  @param _quantity The quantity of tokens to mint.
      *  @param _data Optional extra data passed to the hook.
      *  @return result Abi encoded bytes result of the hook.
      */
-    function beforeMintERC721(address _to, uint256 _quantity, bytes memory _data)
+    function beforeMintERC721(address _caller, address _to, uint256 _quantity, bytes memory _data)
```

Comment:
This is a good change. We should make this change (and make it uniformly) across all Callback functions -- add a `address caller` param and pass `msg.sender`.